src/content/app/species-selector/components/species-search-field/SpeciesSearchField.tsx

```diff
@@ -38,6 +38,8 @@ import CloseButton from 'src/shared/components/close-button/CloseButton';
 import QuestionButton, {
```

Comment:
`SpeciesSearchField` s probably the wrong component to put instructions in. Check out its parent, perhaps?

Reply:
That's done ✅